packages/cli/src/commands/org/config/pull.ts

```diff
@@ -0,0 +1,62 @@
+import {ResourceType} from '@coveord/platform-client';
+import {flags, Command} from '@oclif/command';
+import {Config} from '../../../lib/config/config';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {Snapshot} from '../../../lib/snapshot/snapshot';
+import {SnapshotFactory} from '../../../lib/snapshot/snapshotFactory';
+
+export default class Pull extends Command {
+  public static description = 'Pull resources from an organization';
+
+  public static flags = {
+    target: flags.string({
+      char: 't',
+      description:
+        'The unique identifier of the organization from which to pull the resources. If not specified, the organization you are connected to will be used.',
+      helpValue: 'destinationorganizationg7dg3gd',
+      required: false,
+    }),
+    // TODO: CDX-447: add flag to select resource types to export
+  };
+
+  public static hidden = true;
+
+  @Preconditions(IsAuthenticated())
+  public async run() {
+    const snapshot = await this.getSnapshot();
+
+    await this.refreshProject(snapshot);
+    await snapshot.delete();
+  }
+
+  private async refreshProject(_snapshot: Snapshot) {
+    // TODO: CDX-446: refresh project
```

Comment:
What is "refreshing a project" ?

Reply:
mmh. It's probably not the right word.
I meant refresh/update the project directory.